src/main/java/io/kestra/plugin/serdes/avro/AbstractAvroConverter.java

```diff
@@ -129,6 +130,13 @@ public abstract class AbstractAvroConverter extends Task {
     )
     protected final Property<String> timeZoneId = Property.ofValue(ZoneId.systemDefault().toString());
 
+    @Builder.Default
+    @io.swagger.v3.oas.annotations.media.Schema(
```

Comment:
Import the class properly:

```suggestion
    @Schema(
```